lib/matplotlib/cbook.py

```diff
@@ -850,11 +850,15 @@ def join(self, a, *args):
                     mapping[elem] = set_a
 
     def joined(self, a, b):
-        """Return whether *a* and *b* are members of the same set."""
+        """
+        Return whether *a* and *b* are members of the same set.
+        """
         return (self._mapping.get(a, object()) is self._mapping.get(b))
 
     def remove(self, a):
-        """Remove *a* from the grouper, doing nothing if it is not there."""
+        """
+        Remove *a* from the grouper, doing nothing if it is not there.
+        """
```

Comment:
As above.

Reply:
@timhoffm I manually changed it back in comment https://github.com/matplotlib/matplotlib/pull/29617/commits/08982e9a27d76f4a81d0a9af43e123de1928cdf2